packages/plugins/test/mocha/common.ts

```diff
@@ -407,7 +419,7 @@ export const makeDeployment = async (
 
   const parsedConfigArray = collected.map(
     ({ actionInputs, deploymentInfo }) => {
-      const gasEstimatesArray = actionInputs.map(() => (5_000_000).toString())
+      const gasEstimatesArray = actionInputs.map(() => (2_000_000).toString())
```

Comment:
Necessary to lower this from 5 million to 2 million to make the Rootstock simulation pass (due to the low block gas limit)